README.md

```diff
@@ -41,15 +41,29 @@ Below are the properties allowed by the Changelog Enforcer. These properties are
 * The path to your changelog file. Should be from the perspective of the root directory to `git`. The file being checked for updates must be either an add (`A`) or modified (`M`) status to `git` to qualify as updated. 
 
 `skipLabel` 
-* Default: `Skip-Changelog` 
+* **DEPRECATED - use `skipLabels` input instead**
+* Default: `''`
 * The name of a GitHub label that skips execution of the Changelog Enforcer. This is useful for small changes such as configuration that doesn't need to be reflected in the changelog. By using a label, the developer and any reviewer can agree if the change should be reflected in the changelog, and if not can apply a label. The Changelog Enforcer will re-check if the `labeled` and `unlabeled` event types are specified in the workflow.
 
+`skipLabels` 
+* Default: `'Skip-Changelog'` 
+* List of labels used to skip enforcing of the changelog during a pull request. Each label name is comma seperated and only one label needs to be present for enforcement to be skipped. 
+
+  For example, if `label-1,label-2` was supplied as the `skipLabels`, `label-1` _or_ `label-2` would skip the enforcer. Each label is trimmed for leading and trailing spaces since GitHub labels do not allow for leading or trailing spaces. Thus, the following lists are equivalent:
+  * `label-1,label-2`
+  * `label-1 , label-2`
+  * `label-1  ,label-2`
+
+  The first being the preferred style. 
```

Comment:
Preferred why? I'm with you on preferring the first one, but I'm not sure this library cares. 

Maybe "recommended for readability"?

_Nit_